packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -197,6 +199,7 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         this.deltaManager = dataStoreContext.deltaManager;
         this.quorum = dataStoreContext.getQuorum();
         this.audience = dataStoreContext.getAudience();
+        this.serializer = new FluidSerializer(this, (handle: IFluidHandle) => {});
```

Comment:
I believe this serializer should be removed. I do not think it is used anywhere.
If it is used, I'm not sure I understand what should happen if reference is added when using it.

Reply:
Yes it's not used. I deprecated it recently and plan to remove it when version >= 0.55 -https://github.com/microsoft/FluidFramework/issues/8467